Ananya-Iyer/particleCreator
# first base structure

#### Changes:

Trying to use OpenMaya.api and converting the array to be able to pass it to MParticleSystem which is only compatible with OpenMaya 1

#### To Test:
1. In maya create a mesh object
2. load the plugin
3. run the command `cmds.particleCreator(s=1)` 
4. you can hide the mesh and apply any dynamics to the particle to see the effect.

*NOTE: Details also given in README*
